next.config.js

```diff
@@ -9,6 +9,68 @@ const publicRuntimeConfig = {
     api: process.env.NEXT_PUBLIC_API_HOST || "http://localhost:3000",
     home: process.env.NEXT_PUBLIC_HOME_URL || "http://localhost:3000",
     events: process.env.NEXT_PUBLIC_EVENTS_API || "http://localhost:3334",
+    ipfs: process.env.NEXT_PUBLIC_IPFS_BASE,
+    nft: process.env.NEXT_PUBLIC_NFT_URI,
+    web3Provider: process.env.NEXT_PUBLIC_WEB3_CONNECTION,
+    blockScan: process.env.NEXT_PUBLIC_BLOCKSCAN_LINK
+  },
+  contracts:{
+    settlerToken: process.env.NEXT_PUBLIC_SETTLER_ADDRESS,
+    network: process.env.NEXT_PUBLIC_CONTRACT_ADDRESS,
+    nftToken: process.env.NEXT_PUBLIC_NFT_ADDRESS,
+    networkRegistry: process.env.NEXT_PUBLIC_NETWORK_REGISTRY_ADDRESS,
+    transactionalToken: process.env.NEXT_PUBLIC_TRANSACTION_ADDRESS,
+  },
+  currency:{
+    defaultFiat: process.env.NEXT_PUBLIC_CURRENCY_MAIN,
+    api: process.env.NEXT_PUBLIC_CURRENCY_API,
+    defaultToken: process.env.NEXT_PUBLIC_CURRENCY_ID,
+    conversionList: process.env.NEXT_PUBLIC_CURRENCY_VSLIST,
+  },
+  defaultNetworkConfig: {
+    networkName: process.env.NEXT_PUBLIC_DEFAULT_NETWORK_NAME,
+    allowCustomTokens: process.env.NEXT_PUBLIC_ALLOW_CUSTOM_TOKENS,
+    adminWalletAddress: process.env.NEXT_PUBLIC_ADMIN_WALLET_ADDRESS,
+  },
+  nftUri: process.env.NEXT_PUBLIC_NFT_URI,
+  web3ProviderConnection: process.env.NEXT_PUBLIC_WEB3_CONNECTION,
```

Comment:
We already have these envs in the urls property

Reply:
forgot to remove them